src/krate.rs

```diff
@@ -545,6 +548,16 @@ impl Crate {
         }
     }
 
+    fn documentation_blacklist(url: Option<String>) -> Option<String> {
```

Comment:
Can we give this a more descriptive name? How about `remove_blacklisted_documentation_urls`